community/graphdb-api/src/main/java/org/neo4j/graphdb/temporal/impl/Duration.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.graphdb.temporal.impl;
+
+public class Duration implements org.neo4j.graphdb.temporal.Duration
```

Comment:
This means that we now have *two* implementations of `org.neo4j.graphdb.temporal.Duration`. This seems strange to me, an interface that there is probably no need for, with two separate implementations. Who is helped by this?